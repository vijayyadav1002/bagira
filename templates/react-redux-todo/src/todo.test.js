import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Todo from './todo';

describe('<Todo/>', function () {
    it('should render todo', function () {
        const component = shallow(<Todo/>);
        expect(toJson(component)).toMatchSnapshot();
    });
});