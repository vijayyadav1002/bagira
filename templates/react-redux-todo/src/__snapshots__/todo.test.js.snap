// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Todo/> should render todo 1`] = `
<styled.div>
  <styled.div>
    <styled.h1>
      Things to do
    </styled.h1>
    <Connect(TodoInput) />
    <Connect(TodoList) />
  </styled.div>
</styled.div>
`;
